src/test/java/net/openhft/chronicle/bytes/MappedBytesTest.java

```diff
@@ -88,6 +91,9 @@ public void testMappedFileSafeLimitTooSmall2()
             for (int i = 0; i < 5; i++) {
                 bytesR.write(data);
             }
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
won't Sonar complain about printStackTrace() exposing implementation details or is that only for non-test code?